js/build.js

```diff
@@ -1270,11 +1288,12 @@ Fliplet.Widget.instance('chat', function (data) {
     sortContacts(otherPeople);
 
     // Check if there are selected contacts
-    if (contactsSelected.length) {
-      contactsSelected.forEach(function(contact) {
-        $('[data-contact-id="' + contact.id + '"]').addClass('contact-selected');
-      });
-    }
+    if (!contactsSelected.length) { return; }
+
+    // Create an array of selectors, join it using ',' and then use a single $() selection to .addClass()
```

Comment:
This hasn't been actioned

Reply:
@hcarneiro Just realised `selectorsArray.push('[data-contact-id="' + contact.id + '"]');` could use `_.map()` too, which will be a bit more efficient, but I'm not fussed. You'll just need to remove this comment.